app/javascript/src/components/Dashboard/Settings/ConfirmPasswordFormModal.jsx

```diff
@@ -51,14 +51,12 @@ const ConfirmPasswordFormModal = ({
                 disabled={isSubmitting || !values.password}
                 label="Continue"
                 loading={isSubmitting}
-                size="large"
                 type="submit"
                 onClick={handleSubmit}
               />
               <Button
                 disabled={isSubmitting}
                 label="Cancel"
-                size="large"
                 style="text"
                 onClick={onClose}
               />
```

Comment:
Can you raise an issue and update the https://academy.bigbinary.com/learn-react/pages/formik-forms#:~:text=Formik%20cancel%20button%20type section with this scenario where we shouldn't pass in the `type` as `reset`? The reason is that, we are invoking the reset separately outside of the JSX.

Go through this component to understand.

Reply:
I have added a line about this but yes a proper example will be better.